projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -57,7 +57,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.0" PrivateAssets="None" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.0" PrivateAssets="All" />
```

Comment:
👍 it's very important to make this change.

As it is currently, the `RabbitMQ.Client` package has a _runtime_ dependency on the `Microsoft.CodeAnalysis.FxCopAnalyzers` package.

Reply:
The previous PR was only merged to master. No kittens have been harmed yet :)